app/presenters/hyrax/manifest_components.rb

```diff
@@ -0,0 +1,78 @@
+module Hyrax
+  module ManifestComponents
+    delegate :rendering_ids, to: :solr_document
+
+    def manifest_url
+      manifest_helper.polymorphic_url([:manifest, self])
+    end
+
+    # IIIF rendering linking property for inclusion in the manifest
+    #
+    # @return [Array] array of rendering hashes
+    def sequence_rendering
+      renderings = []
+      if solr_document.rendering_ids.present?
```

Comment:
rendering_ids is not ordered, is this method expecting that it is?

Reply:
No, whilst rendering goes inside the 'sequence' block in the manifest, the rendering has no expectation about order.